dask_cloudprovider/aws/ecs.py

```diff
@@ -401,9 +401,17 @@ class Scheduler(Task):
     See :class:`Task` for parameter info.
     """
 
-    def __init__(self, **kwargs):
+    def __init__(self, scheduler_timeout, scheduler_extra_args=None, **kwargs):
         super().__init__(**kwargs)
         self.task_type = "scheduler"
+        self._overrides = {
```

Comment:
Added this so that the scheduler timeout can be properly set on the scheduler. Previously this argument was part of the task definition.